### ISSUE TITLE
rlwrap is not entirely safe as recommended in the readme

### ISSUE THREAD

**Issue by PsychicCat** [monero-project/monero#1075]
rlwrap ignores password (e.g. '******') input automatically, so it won't store your password. 

but it does not know how to ignore input from prompts like `Specify Electrum seed:`, and probably some other prompts we don't want to have saved in history.

I'm curious how to add a filter to rlwrap if anyone is good with that. 

If filters cannot be added I suggest the README is edited to beware not to use rlwrap if you are going to be entering sensitive data like mnemonic seed, private keys, etc.

**Comment by moneromooo-monero** [comment]
from man rlwrap:

       -a, --always-readline [password_prompt]
              Always  remain  in "readline mode", regardless of command's terminal settings.  Use this option if you want to use rlwrap
              with commands that already use readline.  NB: With this option, rlwrap will echo (and save) passwords,  unless  you  give
              command's  password  prompt  as  an  argument.   The argument is optional; if given, it has to directly follow the option
              without an intervening space.

You might be able to pass several -A options...

**Comment by PsychicCat** [comment]
Yeah, I tried -a but it doesn't seem to work with multiple spaces. I will do some more research on it this weekend and try to find a solution.

for example, this works:

```
rlwrap -apassword: ./simplewallet
```

these do not:

```
rlwrap -aSpecify Electrum seed: ./simplewallet
rlwrap -a="Specify Electrum seed:" ./simplewallet
rlwrap -a"Specify Electrum seed:" ./simplewallet
rlwrap -aSpecify\s+Electrum\s+seed: ./simplewallet
```

**Comment by moneromooo-monero** [comment]
Too bad. I added a note in the README about this then. Thanks for the warning.

**Comment by voidzero** [comment]
I tried this too, and when I specified the following command:

<pre>rlwrap -a'Specify Electrum Seed:' ~/.monero/monero-wallet-cli</pre>

the wallet did run, but the password input was no longer hidden with asterisks, it was shown in plain text.

**Comment by luigi1111** [comment]
Closed via #1083
